uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/staff-leave-report.component.js

```diff
@@ -417,6 +445,14 @@ define([
         });
     }
 
+    /**
+     * Process the list of absence types objects by augmenting and filter them
+     */
+    function processAbsenceTypes () {
+      attachEntitlementsToAbsenceTypes();
+      filterAbsenceTypes();
```

Comment:
why isn't this function used earlier? I don't think we need to load and attach entitlements, and load balance change for absence types that can't be entitled.

Reply:
yes you need to, because part of the filtering criteria is
```js
absenceType.entitlement === 0
```